login.html

```diff
@@ -114,30 +114,31 @@
       submitForm();
      }
   }
-  function submitForm() {
+  function submitForm(postAction) { //Takes either login, register, or resend
```

Comment:
So here I think you ought to pass in the success and error functions as arguments, to avoid having to have an if statement down below (the resend case).